### DISCUSSION TITLE
Was the "Help I'm procrastinating" button removed ?

### DISCUSSION THREAD

**Discussion by ToNyRANDRIAMANANTSOA** [super-productivity/super-productivity#3144] [Q&A] [answered]
Hi there 👋!
In my recent update of SP, I couldn't no longer find the little ⚡ button on the top for showing the anti-procrastination tips. Was it removed from new versions? I mean, I also don't use it so often but sometimes, I find it useful to read the tips inside it, like a quick power pill when in need 😅.

**Comment by alTair21k** [comment]
> In my recent update of SP, I couldn't no longer find the little ⚡ button on the top for showing the anti-procrastination tips.

I'm using the Linux version of the app and I didn't even know about that 😅

**Comment by Jagdfalke** [comment, marked as answer]
The ⚡ is still there in 'Focus Mode':

![image](https://github.com/johannesjo/super-productivity/assets/17673417/b02ecb79-e0bb-47f6-ba1e-4f1b7640ba9c)

**Reply by ToNyRANDRIAMANANTSOA** [reply]
Ah ok Thank you, now I see it in focus mode 😅. I've always used the Pomodoro so I thought it was removed.

**Reply by cubeshit** [reply]
But that doesn't explain why it isn't there anymore in pomodoro mode? Also I couldn't find a mention in the changelogs about removing the button. Its one of my favourite features. It is one of the things that differentiates this app from so many others and shows how much thought and love is put into it.

**Reply by johannesjo** [reply]
@Cubeshit You're right! It should be there for Pomodoro as well! I had plans to merge Pomodoro and focus mode, but was not yet able to execute them. That's why it is missing for pomodoro mode.

**Comment by chantsing** [comment]
Get help was removed from the focus Mode on the version of 15.0.1?
